contracts/yield-source/ATokenYieldSource.sol

```diff
@@ -70,6 +70,13 @@ contract ATokenYieldSource is ERC20Upgradeable, IProtocolYieldSource, AssetManag
   /// @notice Interface for Aave lendingPoolAddressesProviderRegistry
   ILendingPoolAddressesProviderRegistry public lendingPoolAddressesProviderRegistry;
 
+  /// @dev Aave genesis market LendingPoolAddressesProvider's ID
+  /// @dev This variable could evolve in the future if we decide to support other markets
+  uint256 private addressesProviderId = uint256(0);
+
+  /// @dev PoolTogether's Aave Referral Code
+  uint16 private referralCode = uint16(188);
```

Comment:
Make these constants- gas will be cheaper

Reply:
Done: https://github.com/pooltogether/aave-yield-source/pull/23/commits/8073a7c95c708af24caa4def8d76769bcda84db1